stardog-spring-batch/build.gradle

```diff
@@ -85,58 +85,62 @@ signing {
 	required { gradle.taskGraph.hasTask(uploadArchives) }
 }
 
-uploadArchives {
-	repositories {
-		mavenDeployer {
-			beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
-
-			repository(url: "https://oss.sonatype.org/service/local/staging/deploy/maven2/") {
-				authentication(userName: sonatypeUsername, password: sonatypePassword)
-			}
-
-			pom.project {
-				name 'stardog-spring-batch'
-				packaging 'jar'
-				description 'Stardog Spring bindings for using Stardog with the Springframework Batch subproject'
-				url 'https://github.com/clarkparsia/stardog-spring/stardog-spring-batch'
-
-				scm {
-					url 'scm:git://github.com/clarkparsia/stardog-spring.git'
-					connection 'scm:git://github.com/clarkparsia/stardog-spring.git'
-					developerConnection 'scm:git://github.com/clarkparsia/stardog-spring.git'
-				}
-
-				licenses {
-					license {
-						name 'The Apache Software License, Version 2.0'
-						url 'http://www.apache.org/licenses/LICENSE-2.0.txt'
-						distribution 'repo'
-					}
-				}
-
-				developers {
-					developer {
-						id 'albakercp'
-						name 'Al Baker'
-					}
+ uploadArchives {
+ 	repositories {
+ 		mavenDeployer {
+ 			beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
+
+ 			repository(url: "https://oss.sonatype.org/service/local/staging/deploy/maven2/") {
+ 				authentication(userName: sonatypeUsername, password: sonatypePassword)
+ 			}
+
+ 			pom.project {
+ 				name 'stardog-spring-batch'
+ 				packaging 'jar'
+ 				description 'Stardog Spring bindings for using Stardog with the Springframework Batch subproject'
+ 				url 'https://github.com/clarkparsia/stardog-spring/stardog-spring-batch'
```

Comment:
Is this really still under the clarkparsia org? Do we want to migrate it?

Reply:
Actually it's obviously in stardog-union/stardog-spring now. So we should probably change it here